spec/models/monthly_report_spec.rb

```diff
@@ -19,58 +19,40 @@
 require 'rails_helper'
 
 RSpec.describe MonthlyReport, type: :model do
-  it { expect(subject).to respond_to(:user_id) }
-  it { expect(subject).to respond_to(:project_summary) }
-  it { expect(subject).to respond_to(:used_technology) }
-  it { expect(subject).to respond_to(:responsible_business) }
-  it { expect(subject).to respond_to(:business_content) }
-  it { expect(subject).to respond_to(:looking_back) }
-  it { expect(subject).to respond_to(:next_month_goals) }
-  it { expect(subject).to respond_to(:month) }
-  it { expect(subject).to respond_to(:year) }
+  describe 'Validations' do
+    subject { build(:monthly_report) }
 
-  describe '.create' do
-    let(:report) do
-      MonthlyReport.new(
-        user_id: user_id,
-        project_summary: p_summary,
-        used_technology: u_tech,
-        responsible_business: r_busi,
-        business_content: b_cont,
-        looking_back: l_back,
-        next_month_goals: goals,
-        month: month,
-        year: year
-      )
-    end
-    let(:user_id) { 1 }
-    let(:p_summary) { 'hoge' }
-    let(:u_tech) { 'huga' }
-    let(:r_busi) { 'foo' }
-    let(:b_cont) { 'bar' }
-    let(:l_back) { 'mogu' }
-    let(:goals) { 'muga' }
-    let(:month) { 1 }
-    let(:year) { 2020 }
+    it { is_expected.to be_valid }
+    it { is_expected.to validate_presence_of(:user_id) }
+    it { is_expected.to validate_presence_of(:month) }
+    it { is_expected.to validate_inclusion_of(:month).in_range(1..12) }
+    it { is_expected.to validate_presence_of(:year) }
+    it { is_expected.to validate_numericality_of(:year).is_greater_than_or_equal_to(2000) }
```

Comment:
__[rubocop]__
* Line is too long. [91/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)